resources/puppetlabs/lein-ezbake/template/global/ext/cli/reload.erb

```diff
@@ -2,32 +2,34 @@
 set +e
 
 restartfile=<%= EZBake::Config[:restart_file] %>
-timeout=<%= EZBake::Config[:start_timeout] %>
+reload_timeout=${RELOAD_TIMEOUT:-<%= EZBake::Config[:reload_timeout] %>}
+timeout=$reload_timeout
 realname="<%= EZBake::Config[:real_name] %>"
 PIDFILE="/var/run/puppetlabs/${realname}/${realname}.pid"
 
 if [ -r "$restartfile" ];  then
+    cur=$(head -n 1 "$restartfile")
+    initial=$cur
```

Comment:
@shrug - I pushed up b5d48f5, which adds in some extra quoting around variables in the start, stop, and reload scripts.  Let me know if you think that captures the ones you were concerned about.  Thanks!